workspace-grid@mathematical.coffee.gmail.com/utils.js

```diff
@@ -0,0 +1,31 @@
+/***********************************************************************
+ * Copyright (C)      2015 Foivos S. Zakkak <foivos@zakkak.net         *
```

Comment:
If you did not copy any of the code below from another file on this repo then neither me nor Amy Chan should claim any copyright on it

Reply:
I only needed a file to share the same simple functions between two .js files. For me it was more a new file part of the project but I can of course remove your name and the one of Amy of course!

Reply:
The project is open source and can have multiple contributors, since this is your work you are the only one that can claim copyright on it, so if you feel like you can add a copyright notice with your name and contact info